addons/api/addon/serializers/session-recording.js

```diff
@@ -11,4 +11,26 @@ export default class RecordingSerializer extends ApplicationSerializer.extend(
   attrs = {
     connection_recordings: { embedded: 'always' },
   };
+
+  serialize() {
+    let serialized = super.serialize(...arguments);
+    // the post request doesn't expect a payload, so we just delete these
+    delete serialized.retain_until;
+    delete serialized.delete_after;
+    delete serialized.connection_recordings;
+    delete serialized.scope_id;
+    return serialized;
+  }
+
+  normalize(typeClass, hash, ...rest) {
+    const normalizedHash = structuredClone(hash);
+    const normalized = super.normalize(typeClass, normalizedHash, ...rest);
+    // we should not override the ember models errors field,
+    // so we rename the errors field from the api
+
+    const err = normalized.data.attributes.errors;
+    delete normalized.data.attributes.errors;
+    normalized.data.attributes.errors_number = err;
```

Comment:
Is this right? If it's normalized shouldn't we hoist it up to the root like we do with our other attributes?

Reply:
I moved it up

Reply:
tests added